src/item_iterator.rs

```diff
@@ -200,10 +205,14 @@ impl std::fmt::Debug for PublicItem {
 /// we implement `Display` for it.
 impl Display for PublicItem {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        write!(f, "{}", self.tokens)
+        write!(f, "{}", vec_token_to_string(&self.tokens))
     }
 }
 
+pub(crate) fn vec_token_to_string(tokens: &[Token]) -> String {
```

Comment:
I would propose a different name here `tokens_to_string` because implementation details can be left out of names, and because you do not use `Vec` here ;-).

Reply:
Big thanks for review! Good catch. I forgot the rename the function after changing the parameter type. I pushed a fix for it now :)